### DISCUSSION TITLE
New experiments with labelling issues and PRs

### DISCUSSION THREAD

**Discussion by valaparthvi** [redhat-developer/odo#5829] [General]
Issues should have the following labels:
- kind/epic
- kind/user-story
- kind/bug

PRs can have the following labels:
- area/documentation
- area/bug
- area/feature or area/enhancements [T.B.Created]
- area/testing
- area/refactoring
- area/cleanup
and, more.

The change log will be generated based on the PR labels defined above.
This change has been proposed to rid off the long list of labels we have, or at least use the ones that we already have.

**Comment by cdrage** [comment]
Changes would have to be made to three areas if we were to go through with this:

- Github script / changelog script
- Changes to OpenShift bot
- Changes to GitHub issue templates

I'm a big advocate for KEEPING kind/documentation label.

We need to keep this for changes to documentation such as site / README / etc.

**Comment by valaparthvi** [comment]
>I'm a big advocate for KEEPING kind/documentation label.

I think any documentation issue, if it is a feature could be a user story, and we could then assign area/documentation to it to be more precise. Same for the issues related to a bug in the documentation.
